staging/src/k8s.io/cli-runtime/pkg/resource/builder_test.go

```diff
@@ -707,6 +707,37 @@ func TestFilePatternBuilder(t *testing.T) {
 	}
 }
 
+func TestErrorFilePatternBuilder(t *testing.T) {
```

Comment:
I like the table tests, but aren't the two new cases the exact same situations as the ones in the tests you modified above? Maybe I'm missing something, but it seems to me the main difference is the final `if !strings.Contains(err.Error(), pathPattern) {` assertion, which could be added to the table tests.

Reply:
The other are much more granular, these are more higher level and are in a single place, I was considering dropping the other two, but `TestFilePatternBuilderWhenPatternYieldsNoResult` covers 2 cases actually, since it's both glob and a non-existent directory. Where as the non-existent file in my case is just about the file, so I've decided to leave it as us.